Edited README.md to include wiki info, game info, license

mango-gremlin opened the pull request:
## Motivation
Make people aware we have a wiki. Use the wiki.

## What was changed/added
I added a link to the wiki to the readme - ALSO: What exactly are we going to put in the readme? 

**Proposals:**
1. Game Design Finalised Ideas that will serve as guide to implementation (so we have everything in one place)
2. How we want to implement things / guide & documentation to implementation? (or should that stay in the docs?)
3. What else?

Please let me know what you think, multiple things can also work. I think 1 and 2 would both be very helpful, maybe we can split the wiki up a little bit?

---

### Added after comments:
- a banner with the arch-enemy front image ( acting as opening for the github-page)
- short context ( who we are, what we work on )
- Overview of game ( short summary of what this game is about, maybe 3-4 sentences at most? )
- some screenshots ( ofc not yet, we dont have any :D )
- How to run / execute ( ofc not usable yet but giving an example of how to set up the environment )
-  where and how to submit issues (? we wont have much traffic so maybe a little irrelevant)
- used license
- resource / src of assets, libraries used etc ( crediting them accordingly )

ScatteredDrifter commented:
> What exactly are we going to put in the readme?

I would say that the README.md should contain the following:
- a banner with the arch-enemy front image ( acting as opening for the github-page)
- short context ( who we are, what we work on ) 
- Overview of game ( short summary of what this game is about, maybe 3-4 sentences at most? ) 
- some screenshots ( ofc not yet, we dont have any :D ) 
- How to run / execute ( ofc not usable yet but giving an example of how to set up the environment  ) 
- where and how to submit issues (? we wont have much traffic so maybe a little irrelevant)  

further things:
- used license 
- resource / src of assets, libraries used etc ( crediting them accordingly )

mango-gremlin commented:
I added your proposed additions to the readme as well as the godot license (which we still need to include in the credits screen as well i think, but i also added a link to the page where i got the info with the license) - some things i only added the headings for and we can edit them later. 

What does everyone think?
